migrations/1683909884000_inscriptions_per_block.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export function up(pgm: MigrationBuilder): void {
+  pgm.createTable('inscriptions_per_block', {
+    block_height: {
+      type: 'int',
+      primaryKey: true,
+    },
+    inscriptions: {
+      type: 'int',
+      notNull: true,
+    },
+    inscriptions_total: {
+      type: 'int',
+      notNull: true,
+    },
```

Comment:
I suggest renaming these to `block_count` and `accumulated_count` or something more descriptive

Reply:
`inscription_count` and `inscription_count_total`
or just `count` and `count_total` if you rather want to leave out "inscription"

_or something with revealed? `revealed_count`_

Reply:
How about `inscription_count` and `inscription_count_accum`? Something that lets you know the second one is an accumulated count